asv/benchmark.py

```diff
@@ -41,7 +41,7 @@
 import copy
 try:
     import cProfile as profile
-except:
+except Exception:
```

Comment:
We shouldn't capture all exceptions here, just with `ImportError` is enough.

Reply:
let me add this too